### PR TITLE
Add JH PPP

### LINKED ISSUE #9: Add a skeletal PPP
Create a skeletal version of your Project Portfolio Page (PPP). Just the headings are enough. You can write `to be added soon` as placeholders for content.

- [x] JH
- [x] TK
- [x] Jolyn
- [x] ES
- [x] MT

### PR DESCRIPTION
**Pull request by jhchee18**
Add a skletal PPP

### CODE REVIEW

File: docs/team/jhchee18.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: JerHong's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (jolynloh) [inline docs/team/jhchee18.md]:
```suggestion
### Project: CinternS
```

**Review by jolynloh — COMMENTED** [review]
lgtm besides small slip in project title

File: docs/team/jhchee18.md (hunk 1 above)
Comment (jhchee18) [reply docs/team/jhchee18.md]: Thanks for the detailed checking!

**Review by nmtuan2001 — APPROVED** [review]
lgtm

**Pull request merged**